pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
I think that patching is better in this case.

-- 
Sincerely,
Arseniy Seroka



On 18 April 2016 2:22:54 a.m. rardiol <notifications@github.com> wrote:

>> @@ -0,0 +1,23 @@
>> +{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, 
>> libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
>> +
>> +stdenv.mkDerivation rec {
>> +  version = "2016-04-16";
>> +  name = "freeorion-${version}";
>> +
>> +  src = fetchgit {
>> +    url = "https://github.com/freeorion/freeorion";
>
> The prepared source releases avoid this 
> (https://github.com/freeorion/freeorion/releases/download/v0.4.5/FreeOrion_v0.4.5_2015-09-01.f203162_Source.tar.gz), 
> but not with https://github.com/freeorion/freeorion/archive/v0.4.5.tar.gz. 
> I didn't  use either because of the font bug, but maybe if including the 
> .git is bad we can just patch a util/Version.cpp.
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/NixOS/nixpkgs/pull/14775/files/1bad8b0379949b9387e5dd9ad40785b9c5f0e7f8#r59995249